tests/fault_tolerance/deploy/scenarios.py

```diff
@@ -223,12 +223,45 @@ def _create_deployments_for_backend(backend):
     return deployments
 
 
+def _create_moe_deployments_for_backend(backend="vllm"):
```

Comment:
at some point we should consolidate these - maybe we can have a folder passed in to the create_deployments - but need to think about this a little more